R/estimateDiversity.R

```diff
@@ -263,18 +264,18 @@ setMethod("estimateDiversity", signature = c(x="SummarizedExperiment"),
         
         if(!.is_non_empty_character(name) || length(name) != length(index)){
             stop("'name' must be a non-empty character value and have the ",
-                "same length as 'index'.",
-                call. = FALSE)
+                 "same length as 'index'.",
+                 call. = FALSE)
         }
         .check_assay_present(assay.type, x)
         .require_package("vegan")
-
+        
         dvrsts <- BiocParallel::bplapply(index,
-                                        .get_diversity_values,
-                                        x = x,
-                                        mat = assay(x, assay.type),
-                                        BPPARAM = BPPARAM,
-                                        ...)
+                                         .get_diversity_values,
+                                         x = x,
+                                         mat = assay(x, assay.type),
+                                         BPPARAM = BPPARAM,
+                                         ...)
         .add_values_to_colData(x, dvrsts, name)
     }
 )
```

Comment:
indentation seems to be off in these modifications. It should be multiplication of 4 spaces

Reply:
https://contributions.bioconductor.org/r-code.html?q=inden#indentation